Project.toml

```diff
@@ -21,6 +21,7 @@ IntervalSets = "8197267c-284f-5f27-9208-e0e47529a953"
 IterTools = "c8e1da08-722c-5040-9ed9-7db0dc04731e"
 Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
 OffsetArrays = "6fe1bfb0-de20-5000-8ca7-80f57d26f881"
+OhMyREPL = "5fb14364-9ced-5910-84b2-373655c76a03"
```

Comment:
I love OhMyREPL but I guess this should not be here :-)

Reply:
Yes, true.